### ISSUE TITLE
I hate C

### ISSUE THREAD

**Issue by 4Freak** [CyDaRgg/Practise_2semester#2]
#define _CRT_SECURE_NO_WARNINGS
I have some doubts about how relailable your code

**Comment by CyDaRgg** [comment]
i agree with you. So i have corrected some issues which was connected with cheacking for authorization accounts in a binary searching tree.
